packages/create-cloudflare/templates/hello-world-python/py/wrangler.toml

```diff
@@ -0,0 +1,4 @@
+name = "hello-python-worker"
```

Comment:
You should use a placeholder here:

`name = "<TBD>"`

C3 will automatically update this field with the name the user selects in step 1. Otherwise, the user might pick a name like `crimson-sky-1234` but end up with `hello-python-worker` in their `wrangler.toml`

Reply:
```suggestion
name = "<TBD>"
```